packages/x-components/src/x-modules/search/components/__tests__/banners-list.spec.ts

```diff
@@ -1,98 +1,72 @@
-import { Banner } from '@empathyco/x-types';
-import { DeepPartial, Dictionary } from '@empathyco/x-utils';
-import { createLocalVue, mount, Wrapper } from '@vue/test-utils';
-import Vue, {
-  VueConstructor,
-  ComponentOptions,
-  computed,
-  defineComponent,
-  provide,
-  inject,
-  Ref
-} from 'vue';
-import Vuex, { Store } from 'vuex';
+import { DeepPartial } from '@empathyco/x-utils';
+import { mount } from '@vue/test-utils';
+import { computed, defineComponent, provide, inject, Ref, nextTick } from 'vue';
+import { Store } from 'vuex';
 import BaseGrid from '../../../../components/base-grid.vue';
-import { getXComponentXModuleName, isXComponent } from '../../../../components/x-component.utils';
-import { XPlugin } from '../../../../plugins/x-plugin';
-import { RootXStoreState } from '../../../../store/store.types';
+import { LIST_ITEMS_KEY, getXComponentXModuleName, isXComponent } from '../../../../components';
+import { XPlugin } from '../../../../plugins';
+import { RootXStoreState } from '../../../../store';
 import { ListItem } from '../../../../utils/types';
 import { getDataTestSelector, installNewXPlugin } from '../../../../__tests__/utils';
-import { getBannersStub } from '../../../../__stubs__/banners-stubs.factory';
+import { getBannersStub, getResultsStub } from '../../../../__stubs__';
 import BannersList from '../banners-list.vue';
-import { LIST_ITEMS_KEY } from '../../../../components/decorators/injection.consts';
-import { getResultsStub } from '../../../../__stubs__/results-stubs.factory';
+import { searchXModule } from '../../x-module';
 import { resetXSearchStateWith } from './utils';
 
-/**
- * Renders the `BannersList` component, exposing a basic API for testing.
- *
- * @param options - The options to render the component with.
- * @returns The API for testing the `BannersList` component.
- */
-function renderBannersList({
+async function render({
   template = '<BannersList />',
   banners = getBannersStub(),
-  components
-}: RenderBannersListOptions = {}): RendersBannerListAPI {
-  const localVue = createLocalVue();
-  localVue.use(Vuex);
+  components = {}
+} = {}) {
   const store = new Store<DeepPartial<RootXStoreState>>({});
-  installNewXPlugin({ store }, localVue);
 
-  XPlugin.resetInstance();
-
-  resetXSearchStateWith(store, { banners });
   const wrapper = mount(
     {
-      components: {
-        BannersList,
-        ...components
-      },
+      components: { BannersList, ...components },
       template
     },
     {
-      localVue,
-      store
+      global: {
+        plugins: [installNewXPlugin({ store, initialXModules: [searchXModule] })]
+      }
     }
   );
+  resetXSearchStateWith(store, { banners });
+  await nextTick();
 
   const bannersListWrapper = wrapper.findComponent(BannersList);
 
   return {
     wrapper: bannersListWrapper,
```

Comment:
```suggestion
    wrapper: bannersListWrapper: wrapper.findComponent(BannersList);,
```

And delete the const definition above

Reply:
done